kube/services/jobs/config-fence-job.yaml

```diff
@@ -47,7 +47,7 @@ spec:
           - name: PYTHONPATH
             value: /var/www/fence
           - name: CONVERT_OLD_CFG
-            GEN3_CONVERT_OLD_CFG
+            GEN3_CONVERT_OLD_CFG|-value: true-|
```

Comment:
we probably want this to be false/empty by default, this is a one-time thing to convert from local_settings.py to the new config, which I think we've already done in all the commons

Reply:
ok :+1: